index.rst

```diff
@@ -672,43 +701,61 @@ REST API
 To facilitate debugging of prepuller issues, there is a read-only REST API to see the status of a prepull configuration.
 Changing the configuration requires changing the Helm chart or the generated ``ConfigMap`` object and restarting the lab controller.
 
-All of these API calls require ``admin:notebook`` scope.
+All of these API calls require ``admin:jupyterlab`` scope.
 
 ``GET /nublado/spawner/v1/images``
-    Returns a list of known images and their names.
+    Returns an object consisting of the images allowed by the bot users,
```

Comment:
I think this is supposed to be "the images that have been prepulled", or at least I don't know what "allowed by the bot users" means.

Reply:
It returns a list of all images we know about.  `/prepulls` is how we find what should be and has been prepulled.  This endpoint is intended for bot users: this is where you would say "recommended", "latest-daily", or "latest-weekly"....OR you can get a specific tag from the "all images" list, and I've got something to say about the digest a bit lower down.